src/table.js

```diff
@@ -54,6 +54,7 @@ function initialize(
     rows = 11.5, // maximum number of rows to show
     width = {}, // object of column name to width, or overall table width
     multiple = true,
+    select: selectable = true, // is the table selectable?
```

Comment:
I find `selectable` a bit more descriptive than `select` as an option. Thoughts?

Reply:
I have no preference; in the code I've been using selectable because the name is already used by the select function.